src/neptune/internal/backends/hosted_neptune_backend.py

```diff
@@ -981,6 +1006,31 @@ def get_float_series_values(
         except HTTPNotFound:
             raise FetchAttributeNotFoundException(path_to_str(path))
 
+    @with_api_exceptions_handler
+    def _get_float_series_values_proto(
+        self,
+        container_id: str,
+        container_type: ContainerType,
+        path: List[str],
+        offset: int,
+        limit: int,
+    ) -> FloatSeriesValues:
+        params = {
+            "experimentId": container_id,
+            "attribute": path_to_str(path),
+            "limit": limit,
+            "offset": offset,
+        }
+        try:
+            result = self.leaderboard_client.api.getFloatSeriesValuesProto(**params).response().result
```

Comment:
I think this is not going to work with the final version of our Swagger. @asledz what version are we currently using? The one with protobuf-related classes or the clean one?

Reply:
Me and @asledz made some tests and verified it works fine with the final swagger version 🚀